applications/plugins/Xitact/IHPDriver.cpp

```diff
@@ -80,7 +80,7 @@ SOFA_XITACTPLUGIN_API void UpdateForceFeedBack(void* toolData)
     if(myData->xiToolData[0]->lcp_forceFeedback)
     {
         //get tool state for each xitact
-        double pi = 3.1415926535;
+        double pi = M_PI;
```

Comment:
do we really need to keep the pi variable ?

Reply:
Its is very cosmetic at this point :) I would say...so let's avoid aliasing names for the same objects as it proves to be bad.